playground/DayOne.java

```diff
@@ -4,7 +4,7 @@
 import java.util.Scanner;
 
 public class DayOne {
-    final public static String filePath = "../inputs/day1.txt";
+    final public static String filePath = "/inputs/day1.txt";
```

Comment:
See line 25: `sc.close();` , this instruction will not be executed in case of exception.